java.source.base/test/unit/src/org/netbeans/api/java/source/TreePathHandleTest.java

```diff
@@ -422,6 +427,34 @@ public void testLocVar() throws Exception {
         assertNotNull(handle.resolveElement(info));
     }
 
+    public void testVarInstanceMember() throws Exception {
+        try {
+            SourceVersion.valueOf("RELEASE_10"); //NOI18N
```

Comment:
This disables the test on runtime < 10 - shouldn't the nbjavac/parser understand 1.10 even when running on JDK9 ?

Reply:
This does not appear to be wrong, and is actually OK, although probably not absolutely necessary: when running tests with nb-javac (which understands vars), SourceVersion from nb-javac is used, which has RELEASE_10. If we would run the test with vanilla javac on JDK 10, then the constant would exist as well. The only case when the constant does not exist, I think, is when running the tests with vanilla javac (without nb-javac) on JDK 9, which does not understand vars and it would be OK/good to skip the test. But I doubt we would be doing that, so this check is nice, but not absolutely necessary, I think.

Reply:
I have removed this check in latest push.